documentation/navitia.io/source/integration.rst

```diff
@@ -373,21 +376,134 @@ Response example for this request https://api.navitia.io/v1/coverage/fr-idf/phys
+
+    * network : "bobby network"
+    * line : "bobby bus 1"
+    * line : "bobby bus 2"
+    * route : "bobby bus 1 to green"
+    * route : "bobby bus 1 to rose"
+    * route : "bobby bus 2 to yellow"
+    * stop_area : "...
+
+* traveler input: "bobby met"
+
+    * line : "bobby metro 1"
+    * line : "bobby metro 11"
+    * line : "bobby metro 2"
+    * line : "bobby metro 3"
+    * route : "bobby metro 1 to Martin"
+    * route : "bobby metro 1 to Mahatma"
+    * route : "bobby metro 11 to Marcus"
+    * route : "bobby metro 11 to Steven"
+    * route : "...
+
+* traveler input: "bobby met 11" or "bobby metro 11"
 
-Places
-******
+    * line : "bobby metro 11"
+    * route : "bobby metro 11 to Marcus"
+    * route : "bobby metro 11 to Steven"
+
+Parameters
+##########
+
++---------+---------------+----------------------------+------------------------------------+-----------------------------+
+| Required| Name          | Type                       | Description                        | Default value               |
++=========+===============+============================+====================================+=============================+
+| yep     | q             | string                     | The search term                    |                             |
++---------+---------------+----------------------------+------------------------------------+-----------------------------+
+| nop     | type\[\]      | array of string            | Type of objects you want to query  | \[``network``,              |
+|         |               |                            | It takes one the following values: | ``commercial_mode``,        |
+|         |               |                            | \[``network``,                     | ``line``,                   |
+|         |               |                            | ``commercial_mode``,               | ``route``,                  |
+|         |               |                            | ``line``,                          | ``stop_area``\]             |
+|         |               |                            | ``route``,                         |                             |
+|         |               |                            | ``stop_area``\]                    |                             |
++---------+---------------+----------------------------+------------------------------------+-----------------------------+
+
+
+Example
+#######
+
+Response example for : https://api.navitia.io/v1/coverage/fr-idf/pt_objects?q=bus%20ratp%20%39&type[]=line&type[]=route
+
+.. code-block:: json
+
+    {
+    "places": [
+        {
+            {
+
+                "embedded_type": "line",
+                "line": {
+                    ...
+                },
+                "id": "line:RTP:1258386",
+                "name": " RATP Bus 39 (Gare du Nord - Issy Frères Voisin)"
+
+            },
+                    },
+    "links" : [
+        ...
+     ],
+    }
+
+
+Places (/places)
```

Comment:
Same for here, not sure, but maybe it's better to put "Places autocomplete (/places)" ?